modules/node_modules/@frogpond/ccci-stolaf-college/v1/index.js

```diff
@@ -1,24 +1,24 @@
 import koaBody from 'koa-bodyparser'
 
 import Router from 'koa-router'
-import * as atoz from './a-to-z'
-import * as calendar from './calendar'
-import * as contacts from './contacts'
-import * as departments from './departments'
-import * as dictionary from './dictionary'
-import * as faqs from './faqs'
-import * as help from './help'
-import * as hours from './hours'
-import * as jobs from './jobs'
-import * as majors from './majors'
-import * as menus from './menu'
-import * as news from './news'
-import * as orgs from './orgs'
-import * as printing from './printing'
-import * as streams from './streams'
-import * as transit from './transit'
-import * as util from './util'
-import * as webcams from './webcams'
+import * as atoz from './a-to-z/index.js'
```

Comment:
Is there a resolver we could add to map the index.js files?

Reply:
Not when using `node` directly. 

Be glad we're not using TS… you still have to import with .js extensions even though all of your files are .ts